static/src/javascripts/projects/common/modules/navigation/newHeaderNavigation.js

```diff
@@ -80,6 +80,14 @@ define([
                     applyEnhancementsTo(checkbox);
                     checkbox.removeEventListener('click', closeMenuHandler);
                 });
+                if (checkboxId === 'main-menu-toggle') {
+                    // record in Ophan that the menu was opened in a fully expanded state
+                    // i.e. standard JS had not been loaded when menu was first opened
+                    ophan.record({
+                        component: 'main-navigation',
```

Comment:
Hey @obrienm, is the [`Interaction`](https://dashboard.ophan.co.uk/docs/thrift/interaction.html#Struct_Interaction) event the best to use in this case ?